impl/pom.xml

```diff
@@ -70,51 +45,124 @@
+            <artifactId>jakarta.servlet.jsp-api</artifactId>
+            <version>3.0.0-M1</version>
+        </dependency>
+        
+        <!-- Other dependencies -->
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.10.7</version>
+        </dependency>
+        <dependency>
+          <groupId>eclipse</groupId>
+          <artifactId>jdtcore</artifactId>
+          <version>3.1.0</version>
+          <scope>compile</scope>
+        </dependency>
+    </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/java</directory>
+                <includes>
+                    <include>**/*.properties</include>
+                    <include>**/*.xml</include>
+                </includes>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+                <includes>
+                    <include>jakarta.servlet.ServletContainerInitializer</include>
+                </includes>
+                <targetPath>META-INF/services</targetPath>
+            </resource>
+        </resources>
+        
         <plugins>
+            <!-- Sets minimal Maven version to 3.5.4 -->
             <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
-                <version>1.4.3</version>
-                <configuration>
-                    <supportedProjectTypes>
-                        <supportedProjectType>jar</supportedProjectType>
-                    </supportedProjectTypes>
-                    <instructions>
-                        <Bundle-SymbolicName>${bundle.symbolicName}</Bundle-SymbolicName>
-                        <_include>-osgi.bundle</_include>
-                    </instructions>
-                </configuration>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M2</version>
                 <executions>
                     <execution>
-                        <id>bundle-manifest</id>
-                        <phase>process-classes</phase>
+                        <id>enforce-maven</id>
                         <goals>
-                            <goal>manifest</goal>
+                            <goal>enforce</goal>
                         </goals>
+                        <configuration>
+                            <rules>
+                                <requireMavenVersion>
+                                    <version>3.3.0</version>
+                                </requireMavenVersion>
+                            </rules>
+                        </configuration>
                     </execution>
                 </executions>
             </plugin>
+            
+            <!-- Restricts the Java version to 1.8 -->
```

Comment:
Since this is the `/impl` pom, shouldn't this be Java 11?

Reply:
Mark just asked for it to remain on 8, and just make sure it compiles/works on 11. For Jakarta EE 9 I think that's fair.